packages/hint-validate-summary/src/hint.ts

```diff
@@ -0,0 +1,66 @@
+/**
+ * @fileoverview check for summary tag and look for the computedStyle to check whether it is the same as it should be
+ */
+
+import { HintContext } from 'hint/dist/src/lib/hint-context';
+import { IHint } from 'hint/dist/src/lib/types';
+import { StyleEvents, StyleParse } from '@hint/parser-css';
+import meta from './meta';
+import selectorParser = require('postcss-selector-parser');
+
+export default class ValidateSummaryHint implements IHint {
+    public static readonly meta = meta;
+    public readonly requiredAttribute = 'display';
+    public readonly requiredValue = 'list-item';
+
+    public constructor(context: HintContext<StyleEvents>) {
+
+        const validateStyle = ({ ast, resource }: StyleParse) => {
+            const parser = selectorParser();
+
+            ast.walkRules((rule) => {
+
+                if (rule.selector.includes('summary')) {
```

Comment:
Now that we've gotten rid of the `else`, we can reduce nesting later on a bit by converting this to an `if...return`.
```suggestion
                if (!rule.selector.includes('summary')) {
                    return;
                }
```